e2e-tests/cypress/e2e/journeys/insurance/account/sign-in/account-sign-in.spec.js

```diff
@@ -0,0 +1,159 @@
+import { heading, submitButton } from '../../../../pages/shared';
+import partials from '../../../../partials';
+import { signInPage } from '../../../../pages/insurance/account/sign-in';
+import accountFormFields from '../../../../partials/insurance/accountFormFields';
+import {
+  BUTTONS,
+  ORGANISATION,
+  LINKS,
+  PAGES,
+} from '../../../../../../content-strings';
+import { INSURANCE_FIELD_IDS } from '../../../../../../constants/field-ids/insurance';
+import { ACCOUNT_FIELDS } from '../../../../../../content-strings/fields/insurance/account';
+import { INSURANCE_ROUTES as ROUTES } from '../../../../../../constants/routes/insurance';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.ACCOUNT.SIGN_IN.ROOT;
+
+const {
+  START,
+  ELIGIBILITY: { ACCOUNT_TO_APPLY_ONLINE },
+  ACCOUNT: {
+    SIGN_IN: { ROOT: SIGN_IN_ROOT, ENTER_CODE },
+    CREATE: { YOUR_DETAILS },
+  },
+} = ROUTES;
+
+const {
+  ACCOUNT: { EMAIL, PASSWORD },
+} = INSURANCE_FIELD_IDS;
+
+const FIELD_STRINGS = ACCOUNT_FIELDS;
+
+context('Insurance - Account - Sign in - I want to sign in into my UKEF digital service account after completing eligibility, So that I can complete my application for a UKEF Export Insurance Policy', () => {
+  before(() => {
+    cy.navigateToUrl(START);
+
+    cy.submitEligibilityAndStartAccountSignIn();
+
+    const expected = `${Cypress.config('baseUrl')}${SIGN_IN_ROOT}`;
+
+    cy.url().should('eq', expected);
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  it('passes the audits', () => {
+    cy.lighthouse({
+      accessibility: 100,
+      performance: 70,
```

Comment:
Is there a reason why `performance` has been set to `70`?

Reply:
Good shout @abhi-markan just bumped it to 75 like we have in the majority of places - 70 was a copy/paste from account creation which is required.

It's an interesting one because the performance is higher when running locally, but in GHA apparently page load speed takes a little bit longer, so it affects the performance audit check.

75 seems to be fail safe option for GHA - tried having it higher before, but it can be flaky and alter in different PRs.

Reply:
Actually sorry - it needs to be 70 like the "your details" page. This is because some client side MOJ javascript is run  for the password reveal functionality.